### ISSUE TITLE
Provide export templates

### ISSUE THREAD

**Issue by nt1m** [nt1m/devtools-prototyper#50]
Let's say someone wants to start a project out of the prototype. That dev uses ZIP export. He/she has a lot of choices ahead:
- Creating a Firefox OS app
- Creating a Firefox/Chrome add-on
- Create a node.js app
- ...
The add-on should provide templates that can be chosen somehow in the UI, then when a template is chosen, clicking "export as ZIP" will add the appropriate files.

@mdibaiee Any thoughts on a possible UI ?

**Comment by mdibaiee** [comment]
Something similar to the new timing function editor of Firefox:

![export-templates-concept](https://cloud.githubusercontent.com/assets/2807772/9647056/a4e3ea06-51ee-11e5-93a8-44f0253a7bdf.png)

We could default the main button to use the last template.

**Comment by nt1m** [comment]
Yeah, this should work. Maybe we can make it span vertically instead of horizontally ? We can also have an icon-grid like the cubic bezier editor.

**Comment by mdibaiee** [comment]
That's right, vertical should work, too, a tree-like structure.

And oh, you said Firefox OS, I'm working on a FxOS File Manager, join me if you're interested, although the project is still under heavy development, I'll be pushing a lot, things might change and break, you know. :grin: [mdibaiee/hawk](/mdibaiee/hawk)
